build.gradle

```diff
@@ -130,7 +130,7 @@ dependencyLicenses.enabled = false
 // We don't need to following ES naming conventions.
 // see https://github.com/elastic/elasticsearch/blob/5ca6f312058ec8c7fbac072c4648246e7dfb4957/buildSrc/src/main/java/org/elasticsearch/gradle/precommit/NamingConventionsTask.java
 // enable namingConvention check will cause errors like:  "Classes ending with [Tests] must subclass [LuceneTestCase]"
```

Comment:
Should this line be also changed?

Reply:
> Should this line be also changed?

Good catch! Will update the comments for consistency.

Reply:
Done. Thanks!